### ISSUE TITLE
[bug] vchart es5 file can not run in browser

### ISSUE THREAD

**Issue by neuqzxy** [VisActor/VChart#2702]

**Comment by xiaoluoHe** [comment]
check version 1.11.2
![image](https://github.com/user-attachments/assets/f461f7ec-ae2b-4371-83eb-eab2fe62d783)
code: 
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>VChart es5 umd test</title>
  </head>


  <script src="https://sf-unpkg-src.bytedance.net/@visactor/vchart@1.11.2/build/es5/index.js"></script>
  <!-- <script src="https://sf-unpkg-src.bytedance.net/@visactor/vchart@1.11.2/build/index.js"></script> -->

  <body>
    <div id="chart" style="height: 300px"></div>
  </body>

  <script>
    const spec = {
    type: "bar",
    data: [
      {
        id: "barData",
        values: [
          { month: "Monday", sales: 22 },
          { month: "Tuesday", sales: 13 },
          { month: "Wednesday", sales: 25 },
          { month: "Thursday", sales: 29 },
          { month: "Friday", sales: 38 },
        ],
      },
    ],
    xField: "month",
    yField: "sales",
  };
  const vchart = new VChart.default(spec, { dom: 'chart' });
  vchart.renderSync();
  </script>

</html>

```

**Comment by xile611** [comment]
fixed in 1.11.10
